### PR TITLE
Update RemoteID info

### PR DESCRIPTION
**Pull request by Hwurzburg**
updated without fakegps and recent improvements to ArduRemoteID

with a suggestion for the MAVProxy switch  we discussed to allow operator location to be the arming location.....would need some more info in ArduRemoteID as to how to set it in the module code source to reject operator locations set from arming, and that addition to the ArduRemoteID code...

### CODE REVIEW

File: common/source/docs/common-remoteid.rst [hunk 1]
```diff
@@ -26,17 +26,36 @@ Stand-alone devices:
 OpenDroneID
 ===========
 
+.. image:: ../../../images/remoteid.jpg
+    :target: ../../_images/remoteid.jpg
+
+
 ArduPilot provides support for OpenDroneID in firmware versions 4.2 and later via MAVLink to an OpenDroneID compatible module attached to either a serial port or DroneCAN port on the autopilot.
 
 In order to provide tamper protection, a special autopilot firmware must be built and loaded on the autopilot which: enables this feature, prevents loading non-compliant firmware via GCS or normal uploaders, and provides further tamper-proofing.
```
Comment (timtuxworth) [inline common/source/docs/common-remoteid.rst:35]: Make this a bullet list perhaps

File: common/source/docs/common-remoteid.rst [hunk 2]
```diff
@@ -26,17 +26,36 @@ Stand-alone devices:
...
 
-An example of the required modifications for an OpenDroneID firmware version for an autopilot is shown for the `Cube Orange here <https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_HAL_ChibiOS/hwdef/CubeOrange-ODID>`__
+For details on building firmware with OpenDroneID feature, see :ref:`dev:opendroneid` for details.
+
+ArduPilot also has developed firmware for  ESP32-S3 and C3 development boards for an OpenDroneID compatible ID module for attachment by serial MAVLink connection or DroneCAN to the autopilot. A binary for flashing the board, along with instructions, is `available here <https://github.com/ArduPilot/ArduRemoteID>`__ . Note, this firmware is rapidly evolving, so check for updates frequently.
```
Comment (timtuxworth) [inline common/source/docs/common-remoteid.rst:39]: suggest using "Remote ID Module" every time.

File: common/source/docs/common-remoteid.rst [hunk 3]
```diff
@@ -26,17 +26,36 @@ Stand-alone devices:
 OpenDroneID
 ===========
 
+.. image:: ../../../images/remoteid.jpg
+    :target: ../../_images/remoteid.jpg
+
+
 ArduPilot provides support for OpenDroneID in firmware versions 4.2 and later via MAVLink to an OpenDroneID compatible module attached to either a serial port or DroneCAN port on the autopilot.
```
Comment (timtuxworth) [inline common/source/docs/common-remoteid.rst:33]: Would it be clearer to say "flight controller"?

File: common/source/docs/common-remoteid.rst [hunk 4]
```diff
@@ -26,17 +26,36 @@ Stand-alone devices:
...
+
+Mission Planner (must be the latest beta version) has a tab in its DATA view for Drone ID which allows connection of the required external serial GPS to obtain operation location, and setup tabs for UAS and Operator IDs, if required by your local jurisdiction. It also shows Remote ID status.
+The ground station will also have status and pre-arm failure messages displayed if there are problems.
 
-ArduPilot also has developed firmware for an ESP32-S3-M development board for an OpenDroneID compatible ID module for attachment by serial MAVLink connection to the autopilot. A binary for flashing the board, along with instructions, is `available here <https://github.com/ArduPilot/ArduRemoteID>`__ .
+This device can be attached to the autopilot by USB, or serial, or DroneCAN (if external CAN transceiver is attached)
```
Comment (timtuxworth) [inline common/source/docs/common-remoteid.rst:44]: Is an external CAN transceiver required if the flight controller has CAN ports onboard? (which is implied by the statement above at line 33).

File: dev/source/docs/opendroneid.rst [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+    opendroneid set class_eu 1
+    opendroneid set classification_type 1
+    opendroneid set description "TestDrone1"
+    opendroneid set description_type 1
+    opendroneid set operator_id "TestPilot1"
+    opendroneid set operator_id_type 1
+    opendroneid set operator_location_type 0      #note that a value of 1 is usually required in actual operation. This requires using a GCS with its own GPS for testing.
+    opendroneid set rate_hz 1
+
+This sets up the required parameters a user would normally have to enter into the RemoteID system from a GCS.
```
Comment (timtuxworth) [inline dev/source/docs/opendroneid.rst:59]: Probably needs to be "Remote ID" to be consistent.

File: dev/source/docs/opendroneid.rst [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+- Attach autopilot with an OpenDroneID compatible Remote ID module , `like this <https://github.com/ArduPilot/ArduRemoteID>`__ , attached to one of its serial ports to PC and start MAVProxy with the console and map with the option to set the operator location to the vehicle arming location:
+
+.. code::
+
+  MAVProxy.py --console --map
+
+.. note:: this may not be legal in some jurisdictions in actual use, but the OpenDroneID module firmware will reject this assignment with a suitable build option set in its firmware for those jurisdictions. This allows testing on PCs without a GPS attached to determine operator location.
+
+- Make sure the autopilot serial port that the module is attached is setup for 57.6K and MAVLink2
```
Comment (timtuxworth) [inline dev/source/docs/opendroneid.rst:69]: only if using a UART right? Would not apply if using DroneCAN?

**Review by timtuxworth — CHANGES_REQUESTED** [review]
Great start @Hwurzburg - I agree it's important to get it out there. Some minor suggestions.

File: common/source/docs/common-remoteid.rst (hunk 3 above)
Comment (Hwurzburg) [reply common/source/docs/common-remoteid.rst:33]: no...autopilot is used throughout the wiki....many vehicles do not fly

File: common/source/docs/common-remoteid.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-remoteid.rst:35]: its not a series of steps or things to do....general info

File: common/source/docs/common-remoteid.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-remoteid.rst:39]: yep...will do that

File: common/source/docs/common-remoteid.rst (hunk 4 above)
Comment (Hwurzburg) [reply common/source/docs/common-remoteid.rst:44]: added paragraph misplaced.....comment about CAN xcvr was for ARduRemoteID module

File: dev/source/docs/opendroneid.rst (hunk 6 above)
Comment (Hwurzburg) [reply dev/source/docs/opendroneid.rst:69]: right...adding info on that on that line...thanks

File: dev/source/docs/opendroneid.rst [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+- Adding the waf build option, ``--enable-firmware-checking``, will add another tamper proofing check. If enabled, a bootloader will not run firmware with a different board ID. This extends the protection already afforded by using a unique
+
+
+Testing
+=======
+
+- Update MAVProxy/pymavlink to at least 1.8.54 (https://ardupilot.org/mavproxy/docs/getting_started/download_and_installation.html#updating)
+- In the directory you will be running MAVProxy from, create a remoteid.scr file with this content:
+
+.. code-block::
```
Comment (friissoren) [inline dev/source/docs/opendroneid.rst:39]: This code block is not visible on the web page: 
https://ardupilot.org/dev/docs/opendroneid.html#testing

**Review by friissoren — COMMENTED** [review]
I know you already merged this but maybe these problems could be handled in a future PR.

File: common/source/docs/common-remoteid.rst [hunk 8]
```diff
@@ -26,17 +26,35 @@ Stand-alone devices:
 OpenDroneID
 ===========
 
-ArduPilot provides support for OpenDroneID in firmware versions 4.2 and later via MAVLink to an OpenDroneID compatible module attached to either a serial port or DroneCAN port on the autopilot.
+.. image:: ../../../images/remoteid.jpg
+    :target: ../../_images/remoteid.jpg
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-remoteid.rst:30]: fixed

File: dev/source/docs/opendroneid.rst [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+
+    module load OpenDroneID
+
+    opendroneid set UAS_ID_type 1
+    opendroneid set UA_type 1
+    opendroneid set area_ceiling 700
+    opendroneid set area_count 1
+    opendroneid set area_floor -200
+    opendroneid set area_radius 1000
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply dev/source/docs/opendroneid.rst:48]: added notes

File: dev/source/docs/opendroneid.rst (hunk 7 above)
Comment (Hwurzburg) [reply dev/source/docs/opendroneid.rst:39]: fixed

File: dev/source/docs/opendroneid.rst [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+
+.. note:: that the board ID has been changed different from normal firmware, to prevent loading it on a remoteid equipped system by a GCS. And the DID params have been set to read-only to prevent users from changing them and defeating the system.
+
+Building Firmware for Bench Testing and Experimentation
+=======================================================
+
+An experimental firmware version can be built for any autopilot by:
+
+- Adding the line "define AP_OPENDRONEID_ENABLED 1" to its hwdef.dat file, or simply building with the  waf build option, ``--enable-opendroneid``
+- Adding the waf build option, ``--enable-firmware-checking``, will add another tamper proofing check. If enabled, a bootloader will not run firmware with a different board ID. This extends the protection already afforded by using a unique
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply dev/source/docs/opendroneid.rst:30]: fixed

**Pull request merged**